centos.php

```diff
@@ -34,8 +34,9 @@
 $arch = $_GET['arch'];
 $repo = $_GET['repo'];
 
+$known_releases = array_unique(array_merge($stable_releases, $vault_releases, $development_releases));
 $valid_release = $release == '6';
```

Comment:
Is it better to change the default to 7?

Reply:
The centos.php script is used **only** by version 6. Changing the script is not strictly required here, but I'd prefer to use the same variable "known_releases" and keep it aligned to nethserver.php.